Assets/Game/Scripts/Cameras/CameraBounds.cs

```diff
@@ -1,27 +1,30 @@
 ﻿using System;
 using UnityEngine;
 
-namespace GameJammers.GGJ2025.Cameras {
+namespace GameJammers.GGJ2025.Cameras
+{
```

Comment:
Uh oh, I think an auto-formatter may have gone through and reformmatted every file that was opened to change the brackets. Any chance you could disable the auto file formatter for now and revert these files if it isn't too much trouble (I think it will merge conflict with some code I'm working on)?

PS I'm used to inline brackets but I can move to brackets on a new line if that would help reduce merge conflicts?